Lib/test/test_typing.py

```diff
@@ -7315,6 +7353,16 @@ def test_repr(self):
             "typing.Annotated[typing.List[int], 4, 5]"
         )
 
+    def test_dir(self):
+        dir_items = set(dir(Annotated[int, 4]))
+        for required_item in [
+            '__args__', '__parameters__', '__origin__',
+            '__metadata__',
```

Comment:
Note that the `__metadata__` field is not currently documented, but really should be:

- #97797